pkgdb2client/__init__.py

```diff
@@ -705,7 +705,7 @@ def get_packager_package(self, packager, branches=None):
     def get_packages(
             self, pattern='*', branches=None, poc=None, status=None,
             orphaned=False, critpath=None, acls=False, eol=False,
-            page=1, count=False):
+            page=1, limit=250, count=False):
```

Comment:
I checked, and [its 250](https://github.com/fedora-infra/pkgdb2/blob/master/pkgdb2/api/packages.py#L814-L815).  :)